editor-settings.toml

```diff
@@ -81,6 +81,13 @@
 "captions/source+en" = "English"
 "captions/source+es" = "Spanish"
 
+[subtitles.icons]
+# A list of icons to be displayed for languages defined above.
```

Comment:
We could mention that these are totally optional? Arguably that might be obvious, but it might also not be. And in that case, no one feels pressured into thinking up icons when they don't want.

Alternatively comment the list out?

Reply:
I'll update the description